### DISCUSSION TITLE
Support query param "q" for configuring it as a search engine

### DISCUSSION THREAD

**Discussion by ashutoshpw** [open-webui/open-webui#5342] [Ideas]
I use OpenWebUI almost daily, and it would be really great if we could add support for the query parameter 'q' that automatically sends the prompt to a new chat.

This feature would enable us to configure OpenWebUI as a search engine in all major web browsers, allowing us to start conversations directly from the search bar.

e.g.
`https://chat.local.site/?q=Tell me a fun fact` would trigger a new chat with the initial prompt as "Tell me a fun fact"

**Comment by m8dhouse** [comment]
Hi - great news for you then! - this already works as you describe it

https://<yoururl>/?q="tell me a fun fact"&models=gpt-4o-mini

it will open window, start new chat and create icon/title
You can also add a new parameter temporary-chat=true  to have the chat temporary (so not added to history). No other options for the moment.

**Reply by ashutoshpw** [reply]
This is perfect, thanks. For some reason i was trying it on a older version of the installation.
